oci-pipeline-examples/oci-devops-graal-micronaut-deploy-to-instances/README.md

```diff
@@ -63,6 +63,7 @@ Allow dynamic-group <YOUR_DynamicGroup_NAME-2> to use instance-agent-command-exe
 Allow dynamic-group <YOUR_DynamicGroup_NAME-2> to manage objects in compartment <YOUR_COMPARTMENT_NAME>    
 Allow dynamic-group <YOUR_DynamicGroup_NAME-2> to manage objects in compartment <YOUR_COMPARTMENT_NAME>
 Allow dynamic-group <YOUR_DynamicGroup_NAME-2> to manage all-artifacts in compartment <YOUR_COMPARTMENT_NAME>
+Allow dynamic-group <YOUR_DynamicGroup_NAME-2> to read instance-family in compartment <YOUR_COMPARTMENT_NAME>
```

Comment:
Please add the following policy as well
Allow dynamic-group <YOUR_DynamicGroup_NAME-2> to read vnics in compartment <YOUR_COMPARTMENT_NAME>